misc-utils/findmnt.8.adoc

```diff
@@ -131,6 +131,15 @@ Print only pseudo filesystems.
 *--shadow*::
 Print only filesystems over-mounted by another filesystem.
 
+*-Q*, *--filter* _expr_::
+Print only the filesystems that meet the conditions specified by the expr.
++
+This feature is EXPERIMENTAL. See also *scols-filter*(5).
+For exampleinclude xfs having more than 50% free space:
```

Comment:
`For exampleinclude` -> "For example, include"

Reply:
It looks I'm tired. I have made many poor mistakes. Fixed locally.